src/platform/platform_posix.c

```diff
@@ -665,11 +665,11 @@ CxPlatThreadCreate(
 #else // CXPLAT_USE_CUSTOM_THREAD_CONTEXT
 
     //
-    // If pthread_create fails with ENOKEY or ENOENT, then try again without the attribute
+    // If pthread_create fails with an error code, then try again without the attribute
     // because the CPU might be offline.
     //
     if (pthread_create(Thread, &Attr, Config->Callback, Config->Context) &&
```

Comment:
Since we're always going to try the fallback, let's log a Warning, if we did the fallback and it succeeded. And an error if both failed.